sdk/monitor/azure-monitor-opentelemetry/README.md

```diff
@@ -61,7 +61,7 @@ You can use `configure_azure_monitor` to set up instrumentation for your app to
 |-------------------|----------------------------------------------------|----------------------|
 | `connection_string` | The [connection string][connection_string_doc] for your Application Insights resource. The connection string will be automatically populated from the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable if not explicitly passed in. | `APPLICATIONINSIGHTS_CONNECTION_STRING` |
 | `enable_live_metrics` | Enable [live metrics][application_insights_live_metrics] feature. Defaults to `False`. | `N/A` |
-| `logger_name` | The name of the [Python logger][python_logger] under which telemetry is collected. | `N/A` |
+| `logger_name` | The name of the [Python logger][python_logger] under which telemetry is collected. Setting this value is imperative so logs created from the SDK itself are not tracked. | `N/A` |
```

Comment:
This message is a bit weird to me. They can leave it unchecked if they want or even set it specifically to get those logs to. And they can narrow it further with the log level. What is the purpose of this change?

Reply:
It's a common pitfall for customers to not set this value and have the log handler be added on the root logger. In turn, they get an abundance of unneeded logs (and increased costs in which they complain about) because logs from the SDK itself are being captured and exported (sometimes resulting in an infinite loop). Adding this comment makes it so that customers who respect and add the logger name will not run into this extremely common scenario.

Also, not sure what log level has to do with this.

Reply:
Basically, right now, I think this lines reads as "do not capture sdk logs ever". How about something like:
"If you do not wish to track logs from the SDK itself, set the logger name to exclude them."

Reply:
Given how common this misconception is and how almost universally we should expect users to set this value, I'm okay with the current language.